en/upgrades/decimal-upgrade-to-2-03/2-03-changes.rst

```diff
@@ -0,0 +1,183 @@
+2.03 Changelog
+^^^^^^^^^^^^^^
+
+Changes / additions to IATI Standard for version 2.03
+
+Note: Version 2.03 was released on Monday 19th February 2018.
+
+.. contents::
+
+
+.. _2_03_schema_changes:
+
+Schema Changes
+==============
+
+A full diff of all schema changes in 2.03 can be found in the `IATI GitHub <https://github.com/IATI>`__
+
+- `Activities Schema <https://github.com/andylolz/IATI-Schemas/pull/1/commits/97732bdaa5543aa657d2a667f5b23fdfd9ac8f36>`__
+
+- `Organisations Schema <https://github.com/andylolz/IATI-Schemas/pull/1/commits/fa9898cd7f867741aa8a06d90c9b63d30e2d5937>`__
+
+- `Activities and Organisations (Common) Schema <https://github.com/andylolz/IATI-Schemas/pull/1/commits/0b51e6ee4d2231c64857216e5cb675ac8ccbf305>`__
+
+- `Embedded Codelists <https://github.com/IATI/IATI-Codelists/tree/version-2.03dev>`__
+
+- `Non-Embedded Codelists <https://github.com/IATI/IATI-Codelists-Nonembedded/tree/version-2.03dev>`__
```

Comment:
The `2.03-dev` branches will disappear soon, meaning these links will stop working.

Reply:
https://github.com/IATI/IATI-Guidance/commit/fd78daf8c59b311e24964edbcd2488331b8e60fd